tools/scripts-config-react-webpack/README.md

```diff
@@ -52,6 +52,12 @@ Your folder hierarchy should follow
 			"development": "./webpack.config.dev.js",
 			"production": "./webpack.config.prod.js"
 		}
+	},
+	"sentry": {
+		"project": "tmc",
+		"authToken": "xxxxxx",
```

Comment:
it should not be possible to commit a token, so please remove the support of this, only env variables.

Reply:
Right. removed.